### PR TITLE
enable s3 tests in internal gitlab runners

### PR DESCRIPTION
**Pull request by guihao-liang**

### CODE REVIEW

File: src/python/turicreate/test/test_sframe_s3.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 
 @pytest.mark.skipif(
-    os.environ.get("TURI_ENABLE_SF_S3") is None,
+    os.environ.get("TC_ENABLE_S3_TESTS", "0") == "1",
```
Comment (hoytak) [inline src/python/turicreate/test/test_sframe_s3.py]: Doesn't this skip it if it's 1?

File: src/python/turicreate/test/test_sframe_s3.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def setup_class(self):
             # force clean in case same tempdir is reused without cleaning
             try:
                 shutil.rmtree(tmp_folder)
-            except FileNotFoundError:
+            except OSError:
```
Comment (hoytak) [inline src/python/turicreate/test/test_sframe_s3.py]: Just change this to :
```
except Exception, e:
     warning.warn("Error raised while cleaning up %s: %s" % (tmp_folder, str(e)))
```

File: src/python/turicreate/test/test_sframe_s3.py (hunk 1 above)
Comment (guihao-liang) [reply src/python/turicreate/test/test_sframe_s3.py]: do you set that to 1 to skip? I though 1 means to turn it on.

File: src/python/turicreate/test/test_sframe_s3.py (hunk 2 above)
Comment (guihao-liang) [reply src/python/turicreate/test/test_sframe_s3.py]: that would be good.

File: src/python/turicreate/test/test_sframe_s3.py (hunk 1 above)
Comment (guihao-liang) [reply src/python/turicreate/test/test_sframe_s3.py]: ah, may bad.

**Pull request merged**